app/pages/Auction/RepairBid.js

```diff
@@ -4,6 +4,7 @@ import PropTypes from 'prop-types';
 import {consensus} from 'hsd/lib/protocol';
 import walletClient from '../../utils/walletClient';
 import * as names from '../../ducks/names';
+import { showError } from '../../ducks/notifications';
```

Comment:
I don't think this is necessary, `showError` is connected a few lines down

Reply:
It seems like you forgot to import if before connecting. 
Without it, there is an console error, about `showError` not being defined.

Reply:
ok cool thank you